src/Compilers/CSharp/Test/Semantic/Semantics/DeconstructionTests.cs

```diff
@@ -2620,5 +2634,91 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_TypeVarNotFound, "var").WithLocation(6, 15)
             );
         }
+
+        [Fact]
+        public void MixedDeconstruction_01()
```

Comment:
Sure, but I will have to move them to CodeGenDeconstructTests where the infrastructure exists for that.

---
In reply to: [90169982](https://github.com/dotnet/roslyn/pull/15548#discussion_r90169982) [](ancestors = 90169982)